posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Deploy Java applications to IBM Cloud Code Engine using the Paketo Buildpack for Liberty"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars.githubusercontent.com/u/29545669?v=4
+author_github: https://github.com/kevin-ortega
+seo-title:  Deploy Java applications to IBM Cloud Code Engine (ICCE) using the Paketo Buildpack for Liberty - OpenLiberty.io
+seo-description: Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty
+blog_description: "Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty"
+open-graph-image: https://openliberty.io/img/blog/code_engine_build_step_details.png
```

Comment:
My suggestion next time is to try using a relative path for the `open-graph-image` value. As of right now, the image used here is not on `openliberty.io` yet.

I usually will test the SEO fields (like the `og:image`) in Slack to see the metadata that gets pulled into preview cards. Here is an example of me DMing myself the link to test the Slack's URL preview cards. Notice an image is missing.
<img width="1044" alt="image" src="https://user-images.githubusercontent.com/31117513/201743393-7df75c0a-d709-44e6-aadf-0d95ccd1c1e1.png">

Reply:
Woops, I did not give a suggestion on a relative path. Here is what I would of tried `/img/blog/code_engine_build_step_details.png`.

Reply:
from my recollection, relative paths use to not work for this.... but maybe things have changed or I'm misremembering